esp/src/eclwatch/ESPResult.js

```diff
@@ -77,6 +77,9 @@ define([
             }
         },
         preProcessResponse: function (response, request) {
+            if (response.Total == -1 || response.Total === 9223372036854776000 || response.Total === Number.MAX_VALUE) {
```

Comment:
Alas no - JS starts rounding before that (see the trailing 000s) - In practice the "-1" condition and the "MAX_VALUE" will never be hit, I just put them there as a way of commenting (and just in case the server does start to return either of those).